alu_u32/src/add/mod.rs

```diff
@@ -113,6 +116,16 @@ where
         let a = b + c;
         state.mem_mut().write(clk, write_addr, a, true);
 
+        // Record the output of the operation (a) in the range check chip
+        for i in 0..MEMORY_CELL_BYTES {
+            state
```

Comment:
Maybe a helper on `MachineWithRangeChip` would be nice to encapsulate this bit of logic